planet_helpers.go

```diff
@@ -72,7 +72,8 @@ func regionPlanets(w http.ResponseWriter, r *http.Request) ([]*ent.Planet, error
 		All(r.Context())
 
 	if _, ok := err.(*ent.ErrNotFound); ok {
-		return nil, newNotFoundError(err)
+		p := make([]*ent.Planet, 0)
+		return p, newNotFoundError(err)
```

Comment:
Rereading this, I don't think `All(r.Context)` actually returns a not found error. So this whole if block can be removed.

But if it did, what I meant was:
```golang
return nil, nil
```
So you're returning a nil-slice (which Go handles as if you passed an empty slice) and no error.

Reply:
Ok